circuit_maintenance_parser/parsers/seaborn.py

```diff
@@ -0,0 +1,102 @@
+"""Telstra parser."""
+import logging
+import re
+from typing import Dict, List
+
+from dateutil import parser
+from bs4.element import ResultSet  # type: ignore
+
+from circuit_maintenance_parser.errors import ParserError
+from circuit_maintenance_parser.parser import CircuitImpact, Html, Impact, Status, Subject
+
+# pylint: disable=too-many-branches
+
+
+logger = logging.getLogger(__name__)
+
+
+class SubjectParserSeaborn1(Subject):
+    """PArser for Seaborn subject string."""
+
+    def parse_subject(self, subject, data_base):
+        data = data_base.copy()
+        try:
+            search = re.search(r".+\[## ([0-9]+) ##\].+", subject)
+            if search:
+                data["account"] = search.group(1)
+            return [data]
+
+        except Exception as exc:
+            raise ParserError from exc
+
+
+class HtmlParserSeaborn1(Html):
+    """Notifications Parser for Seaborn notifications."""
```

Comment:
following the patter you introduced in the previous parser, an example of the HTML data would be great!